src/app/game/game-result/game-result.component.scss

```diff
@@ -109,26 +157,33 @@
 
 .score {
   color: var(--White, #fbfbfb);
-  font-family: AeonikLight;
+  font-family: AeonikRegular;
   font-size: 24px;
   font-style: italic;
-  font-weight: 300;
+  font-weight: 500;
   line-height: normal;
 }
+
+.loss {
+  color: #de1f1f;
+}
+
+.win {
+  color: #68d466;
+}
 .result-word {
   color: var(--White, #fbfbfb);
   text-align: center;
   font-family: AeonikLight;
   font-size: 24px;

```

Comment:
As the font sizes are reused within this file, yet differ from the globally defined ones, it could be nice if you made them into local variables! ☺️

`* {`
`--resultFontSizeSmall: 24px;`
`}`
``
`(...)`
``
`.result-word{`
`font-size: var(--resultFontSizeSmall);`
`}`